backend/inspirehep/records/models.py

```diff
@@ -309,3 +309,37 @@ class WorkflowsRecordSources(db.Model, Timestamp):
         JSONB(),
         default=lambda: dict(),
     )
+
+class JournalLiterature(db.Model):
```

Comment:
if you're adding this you need a database schema migration (alembic), otherwise it won't work